### PR TITLE
Remove scvmm and hyperv since core dropped it

### PR DESCRIPTION
**Pull request by jrafanie**
See: https://github.com/ManageIQ/manageiq/pull/22279

I noticed this while looking to add zeitwerk in gems pending, see: https://github.com/ManageIQ/manageiq-gems-pending/pull/551

I used the following to find these:
`git grep -iE "win_rm|winrm|scvmm|hyperv"`

Related PRs:
https://github.com/ManageIQ/manageiq/pull/22413
https://github.com/ManageIQ/manageiq-smartstate/pull/172 (this one)
https://github.com/ManageIQ/manageiq-gems-pending/issues/556

### CODE REVIEW

**Comment by Fryguy** [comment]
😍 

Can manageiq-gems-pending itself be removed after this?

**Comment by jrafanie** [comment]
@miq-bot cross-repo-test including manageiq#22413, manageiq-smartstate#172, manageiq-gems-pending#556

**Comment by jrafanie** [comment]
@miq-bot cross-repo-test manageiq-smartstate#172 including manageiq#22413, manageiq-gems-pending#556

**Comment by jrafanie** [comment]
Cross repo results for the 3 related PRs.  I think they're ready to be merged.  Removing WIP.
- [ ] https://github.com/ManageIQ/manageiq/pull/22413 
  - [x] 🟡 https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/764
    - 2 failures
      - ui service, always seems to fail with cross repo
      - fix: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/447
- [ ] https://github.com/ManageIQ/manageiq-gems-pending/pull/556
  - [x] 🟢 https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/767 
- [ ] https://github.com/ManageIQ/manageiq-smartstate/pull/172
  - [x] 🟢 https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/768

**Comment by agrare** [comment]
@jrafanie are the `MSVS` disk modules still used?  It seems like if we don't scan hyperv anymore we wouldn't need those modules

**Comment by jrafanie** [comment]
> @jrafanie are the `MSVS` disk modules still used? It seems like if we don't scan hyperv anymore we wouldn't need those modules
> 
> image
> 
> You deleted the hyperv connections out of these but I'm not sure when they would be used now

All good questions.  I'm afraid of peeling back this onion.  I will though, because the potential for 🔴🔴🔴 is great.

File: lib/disk/DiskProbe.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module DiskProbe
     p.chomp!(".rb")
   end
   # Reorder known probes for optimization
-  %w(VixDiskProbe VMWareDiskProbe RhevmDiskProbe VhdxDiskProbe).reverse_each do |probe|
+  %w(VixDiskProbe VMWareDiskProbe RhevmDiskProbe).reverse_each do |probe|
```
Comment (jrafanie) [inline lib/disk/DiskProbe.rb:12]: sorry, "technically", I added a line here

**Comment by jrafanie** [comment]
Ok, updated.  Now with even more deletions!

File: lib/disk/DiskProbe.rb (hunk 1 above)
Comment (kbrock) [reply lib/disk/DiskProbe.rb:12]: oh so close 🔥

**Review by agrare — APPROVED** [review]

**Pull request merged**